.github/workflows/static-analysis.yaml

```
@@ -2,6 +2,7 @@ name: Static Code Analysis
 
 on:
   pull_request: { }
+  merge_group: { }
```

```suggestion
  merge_group:
    types: [checks_requested]
```

per https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#merge_group